packages/tooltip/src/Tooltip.spec.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import 'jest-dom/extend-expect';
+import { render, fireEvent, cleanup } from '@testing-library/react';
+import Tooltip from '.';
+
+afterAll(cleanup);
+
+describe('packages/Tooltip', () => {
+  const buttonText = 'trigger button';
+
+  describe('when uncontrolled', () => {
+    const triggerEvent = 'click';
+
+    const { getByText, getByTestId } = render(
+      <>
+        <div>backdrop content</div>
+        <Tooltip
+          trigger={<button>{buttonText}</button>}
```

Comment:
Can we get a test for when the trigger is a function.

And maybe one where it's a class? Not sure if we want to worry about this right now.

Reply:
👍